### PR TITLE
[Krypto-9] project cleanup

### PR DESCRIPTION
**Pull request by Francmape**
Please check these project cleanup file

### CODE REVIEW

**Comment by Wamae** [comment]
@Francmape, these are too many files to review. Could you keep it to a maximum of 10 files per PR? Kindly break down this PR into bite-size pieces.

**Comment by Francmape** [comment]
@Wamae, These are the files that I only changed the package name on the first line.

**Comment by Wamae** [comment]
Screenshot from 2022-04-27 13-45-54
Yellow shows modified and green are new files.

**Comment by Wamae** [comment]
Please resolve the conflicts.

**Comment by Francmape** [comment]
I think I messed up while Resolving conflicts. I can't find of my files

File: app/src/main/java/ke/co/svs/mykrypto/KryptoApp.kt [hunk 1]
```diff
@@ -14,8 +21,18 @@ class KryptoApp : Application() {
...
             androidContext(this@KryptoApp)
+
             // declare modules
-            modules(listOf(retrofitModule))
+            modules(
+                apiModule,
+                retrofitModule,
+                repositoryModule,
+                viewModelModule,
+                databaseModule
```
Comment (Wamae) [inline app/src/main/java/ke/co/svs/mykrypto/KryptoApp.kt]: databaseModule does not exist, it makes build fail.

File: app/src/main/java/ke/co/svs/mykrypto/cryptos/data/data_source/CryptoDao.kt [hunk 2]
```diff
@@ -1,12 +1,23 @@
 package ke.co.svs.mykrypto.cryptos.data.data_source
 
-import androidx.room.Dao
-import androidx.room.Query
+import androidx.room.*
 import ke.co.svs.mykrypto.domain.model.Crypto
```
Comment (Wamae) [inline app/src/main/java/ke/co/svs/mykrypto/cryptos/data/data_source/CryptoDao.kt:4]: Crypto model does not exist.

File: app/src/main/java/ke/co/svs/mykrypto/ui/screens/home/HomeScreenViewModel.kt [hunk 3]
```diff
@@ -1,11 +1,84 @@
...
+
+        //Fetching from Network
+        fetchCryptos()
+
+    }
+
+    fun onEvent(event: HomeScreenCallbackEvents) {
+        when (event) {
+            is HomeScreenCallbackEvents.OnCryptoItemClick -> {
+                Log.i(TAG, "CLICKED")
```
Comment (Wamae) [inline app/src/main/java/ke/co/svs/mykrypto/ui/screens/home/HomeScreenViewModel.kt:45]: Use timber for logging: https://github.com/JakeWharton/timber

**Review by Wamae — APPROVED** [review]

**Pull request merged**